elf2uf2/elf2uf2.cpp

```diff
@@ -118,13 +169,26 @@ int realize_page(std::shared_ptr<std::iostream> in, const std::vector<page_fragm
     return 0;
 }
 
+/**
+ * @brief Checks if a specific address is mapped.
+ * 
+ * @param pages Map of pages with their fragments.
+ * @param addr Address to check.
+ * @return true if the address is mapped, false otherwise.
+ */
 static bool is_address_mapped(const std::map<uint32_t, std::vector<page_fragment>>& pages, uint32_t addr) {
     uint32_t page = addr & ~(UF2_PAGE_SIZE - 1);
     if (!pages.count(page)) return false;
-    // todo check actual address within page
+    // TODO: Check actual address within page
     return true;
 }
 
+/**
+ * @brief Generates an absolute UF2 block. The absolute block is required to work around Errata E10 in the RP2040 datasheet.
```

Comment:
I suspect you meant the RP2350 datasheet here, not RP2040?
https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf
https://datasheets.raspberrypi.com/rp2350/rp2350-datasheet.pdf